app/assets/stylesheets/registered_applications.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the registered_applications controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
```

Comment:
Consider deleting these unused skeleton asset files (`registered_applications.scss/coffee`), they're not needed and can end up making the asset subdirs feel cluttered.

These assets are generated by `rails g ...`.

To stop these and other unused files from being added to the repo in future, carefully review `git status` output before `git add`-ing, be ready to `rm unwanted/file`, and be selective with `git add` (`git add path/to/dir path/to/file` vs `git add .`).